scripts/build.py

```diff
@@ -671,7 +671,7 @@ def build_using_webpack(config_path):
 
     python_utils.PRINT('Building webpack')
     managed_webpack_compiler = servers.managed_webpack_compiler(
-        config_path=config_path, max_old_space_size=2400)
+        config_path=config_path, max_old_space_size=4096)
```

Comment:
Will this affect the developer's expirence in any way?

Reply:
No it shouldn't, if the value is 2400 there is a risk that the app will fail to build.